src/pages/index.js

```diff
@@ -0,0 +1,311 @@
+      setTimeout(() => {
+        addNewCardSubmitButton.textContent = "Add";
+      }, 1000);
+    });
+}
+
+function handleCardLikes(cardData) {
+  if (!cardData._like) {
+    //if not liked
+    console.log(cardData._like);
+    api
+      .cardLike(cardData._id)
+      .then(() => {
+        cardData.updateLike(true); //call card.js like method
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  } else if (cardData._like) {
+    //if liked
+    api
+      .cardUnlike(cardData._id)
+      .then(() => {
+        cardData.updateLike(false);
+      })
+      .catch((err) => {
+        console.error(err);
+      });
+  }
+}
+
+function handleDeleteModalOpen(cardData) {
+  deleteConfirm.handleFormSubmit(() => {
+    //pass handleCardDelte to handleFormSubmit method to set event listeners independently
+    handleCardDelete(cardData);
+  });
+  //open delete modal
+  deleteConfirm.openPopup();
+}
+
+function handleCardDelete(cardData) {
+  //pass in card data
+  console.log(cardData);
+  deleteModalButton.textContent = "Deleting...";
+  api
+    .deleteCard(cardData._id)
+    .then(() => {
+      cardData.removeCard(cardData._id); //call card.js delete
+      deleteConfirm.closePopup();
+    })
+    .catch((err) => {
+      console.error(err);
+    })
+    .finally(() => {
+      setTimeout(() => {
+        deleteModalButton.textContent = "Delete";
+      }, 1000);
+    });
+}
+
+function createCard(cardData) {
+  //function to create new card
+  const card = new Card(
+    cardData,
+    "#card-template",
+    handleImageClick,
+    handleDeleteModalOpen,
+    handleCardLikes
+  ); //instantiate card class
+  const cardElement = card.viewCard(); //call viewCard of card class to create
+  return cardElement; //return card
+}
+
+function renderCard(cardData) {
+  //function to render new card to page
+  const cardElement = createCard(cardData); //create card;
+  section.addItem(cardElement);
+}
+
+function handleAvatarModalOpen() {
+  avatarModal.classList.add("modal_open");
+}
+
+function handleAvatarModalClose() {
+  avatarModal.classList.remove("modal_open");
+}
+
+///////////////////////////////////////////// EVENT LISTENERS ///////////////////////////////////////////////////////
+/* cardDeleteButton.addEventListener("click", () => {
+  console.log("Delete");
+}); */
+
+//////////////////////////////////////////// Profile Modal ////////////////////////////////////////////////////////////
+profileEditButton.addEventListener("click", () => {
+  const userInfo = user.getUserInfo(); //get user info
+  profileTitleInput.value = userInfo.name; //pre set name
+  profileDescriptionInput.value = userInfo.job; //pre set job
+  profilePopup.openPopup(); //open popup
+});
+
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+avatarEditButton.addEventListener("click", () => handleAvatarModalOpen());
+avatarModalCloseButton.addEventListener("click", () =>
```

Comment:
`avatarModalCloseButton.addEventListener("click", () =>
`

Please, delete the close icon handler because everyt close icon is handled by `Popup`